usr/lib/common/template.c

```diff
@@ -999,10 +1037,16 @@ CK_RV template_set_default_common_attributes(TEMPLATE *tmpl)
     label_attr->ulValueLen = 0; // empty string
     label_attr->pValue = NULL;
 
+    unique_id_attr->type = CKA_UNIQUE_ID;
+    unique_id_attr->ulValueLen = sizeof(64);
```

Comment:
ok, the ulValueLen is sizeof(64), but what type does 64 have? It's just a number ...

Reply:
Should just be 64 i.e. 2*UNIQUE_ID_LEN, dont know why comiler doesnt complain ..

Reply:
done